storybooknative/src/InputField/index.js

```diff
@@ -0,0 +1,32 @@
+// @flow strict
+
+import * as React from "react";
+import { TextInput } from "react-native"; // eslint-disable-line no-restricted-imports
+
+import Text from "../Text";
+import styled from "../styled";
+
+type Props = {|
+  +placeholder?: string,
+  +label?: string,
+  +defaultValue?: string,
+  +autoFocus?: boolean,
+  +onChangeText?: (text: string) => void,
+  +keyboardType?: "email-address" | "numeric" | "number-pad",
+  +secureTextEntry?: boolean,
+  +maxLength?: number,
+|};
+
+const StyledTextInput = styled(TextInput, props => {});
+
+const InputField = (props: Props) => {
+  const { label, ...rest } = props;
+  return (
+    <React.Fragment>
+      {label && <Text>{label}</Text>}
```

Comment:
I think it was better to keep the label as being of type `Text`, because then we could use `Translate` component and the flow types would match.

Reply:
And so now you can remove the Text tags I guess.